app/assets/javascripts/select-kit/addon/components/select-kit.js

```diff
@@ -207,13 +207,14 @@ export default Component.extend(
     didReceiveAttrs() {
       this._super(...arguments);
 
-      Object.keys(this.selectKitOptions).forEach((key) => {
+      const mergedOptions = Object.assign({}, ...this.selectKitOptions);
```

Comment:
@jjaffeux I rescind my approval! 😂 That doesn't seem right, how does it not crash and burn? 😅